examples/transcription/README.md

```diff
@@ -16,19 +16,12 @@ the different models on different accents and English fluency levels.
 The result of running Zeno Build will be an interface where you
 can browse and explore the results. See an example below:
 
-- [Browsing Interface](https://zeno-ml-transcription-report.hf.space)
-- [Textual Summary](report/)
+- [Browsing Interface](https://hub.zenoml.com/project/cabreraalex/Audio%20Transcription%20Accents/explore)
+- [Textual Summary](https://hub.zenoml.com/report/cabreraalex/Audio%20Transcription%20Report)
```

Comment:
Should we start using the `open with zeno` button here? Also since I just looked at the button should we capitalize Zeno?

Reply:
Great idea!